src/js/components/agencyV2/visualizations/StatusOfFundsChart.jsx

```diff
@@ -336,48 +329,13 @@ const StatusOfFundsChart = ({
             .attr("y1", isMobile ? chartHeight + 1040 : horizontalBorderYPos())
             .attr("x2", isLargeScreen ? chartWidth + 330 : chartWidth + 100)
             .attr("y2", isMobile ? chartHeight + 1040 : horizontalBorderYPos());
-        // append labels for legend below chart
-        svg.append("circle")
-            .attr("cx", isLargeScreen ? -130 : 180)
-            .attr("cy", isMobile ? chartHeight + 1080 : legendObligationsYPos())
-            .attr("r", 6)
-            .style("fill", "#2B71B8");
-        svg.append("circle")
-            .attr("cx", isLargeScreen ? -130 : 375)
-            .attr("cy", isMobile ? chartHeight + 1150 : legendResourcesYPos())
-            .attr("r", 6)
-            .style("fill", "#BBDFC7");
-        svg.append("text")
-            .attr("x", isLargeScreen ? -115 : 195)
-            .attr("y", isMobile ? chartHeight + 1081 : legendObligationsYPos() + 1)
-            .text(`FY${fy[2]}${fy[3]} Obligations`)
-            .attr('class', 'y-axis-labels')
-            .style('fill', '#555')
-            .attr("alignment-baseline", "middle");
-        svg.append("text")
-            .attr("x", isLargeScreen ? -115 : 390)
-            .attr("y", isMobile ? chartHeight + 1151 : legendResourcesYPos() + 1)
-            .text(`FY${fy[2]}${fy[3]} Total Budgetary Resources`)
-            .attr('class', 'y-axis-labels')
-            .style('fill', '#555')
-            .attr("alignment-baseline", "middle");
-        if (isNegative) {
-            svg.append('line')
-                .attr('transform', tickMobileXAxis)
-                .style("stroke", "#aeb0b5")
-                .style("stroke-width", 3)
-                .attr("x1", x(0))
-                .attr("y1", 0)
-                .attr("x2", x(0))
-                .attr("y2", isLargeScreen ? chartHeight + 500 : chartHeight + 4);
-        }
```

Comment:
Small merge conflict issue here - we want to keep the if (isNegative) conditional for dev-8331, that is not part of the legend